### PR TITLE
example: get ten users with highest priority

### PR DESCRIPTION
**Pull request by ValeriyaSinevich**

### CODE REVIEW

File: examples/get_ten_users_with_highest_reputation.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                     auth=auth
+                     )
+token = json.loads(resp.text)['access_token']
+
+def get_users(tok):
+    page = 1
+    reputations = []
+    logins = []
+    api_url = 'https://stepic.org/api/users?page=' + str(page);
+    result = json.loads(requests.get(api_url, headers={'Authorization': 'Bearer ' + tok}).text)
```
Comment (psviderski) [inline examples/get_ten_users_with_highest_reputation.py]: A `Response` object has a special `json()` method that performs this `json.loads(...text)` trick.

File: examples/get_ten_users_with_highest_reputation.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    while(result['meta']['has_next'] == True):
```
Comment (psviderski) [inline examples/get_ten_users_with_highest_reputation.py]: It's unnecessary to compare a boolean value with `True`, just put the condition only:
```
while result['meta']['has_next']:
    ...
```

File: examples/get_ten_users_with_highest_reputation.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        profiles = result['users']
+        for profile in profiles:
+            reputations.append(profile['reputation'])
+            name = profile['first_name'] + " " + profile['last_name']
+            logins.append(name)
+            page += 1
+            api_url = 'https://stepic.org/api/users?page=' + str(page);
+            result = json.loads(requests.get(api_url, headers={'Authorization': 'Bearer '+ tok}).text)
```
Comment (psviderski) [inline examples/get_ten_users_with_highest_reputation.py]: There is a way not to duplicate this get request and only have it inside the loop. Please see another pull request for an example: https://github.com/StepicOrg/Stepic-API/pull/8/files

**Comment by mehanig** [comment]
Looks good, merged

**Pull request merged**